### DISCUSSION TITLE
How to manually update dnscrypt version on my Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by persian85** [DNSCrypt/dnscrypt-proxy#1879] [Q&A] [answered]
So if I just copy the newest version of dnscrypt files to my installed folder is it the correct way to update it?

**Comment by jedisct1** [comment, marked as answer]
See https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation#upgrading

**Reply by persian85** [reply]
Thank you
